include/fastrtps/types/TypesBase.h

```diff
@@ -364,7 +368,10 @@ class TypeFlag
         m_TypeFlag = std::move(x.m_TypeFlag);
         return *this;
     }
-
+				
+				std::bitset<16> getM_typeFlag() {return m_TypeFlag; }
```

Comment:
Tabs and the body of the function must start always in the next line, including the opening bracket.

Reply:
done